src/org/zaproxy/zap/extension/httpsinfo/ExtensionHttpsInfo.java

```diff
@@ -101,7 +101,16 @@ private MenuEntry getPopupMsgMenu() {
 	
 	protected TabbedPanel2 getHttpsInfoTabsPanel() {
 		if (httpsInfoTabsPanel == null) {
-			httpsInfoTabsPanel = new TabbedPanel2();
+			httpsInfoTabsPanel = new TabbedPanel2() {
+				private static final long serialVersionUID = -1422894398829082869L;
+
+				@Override
+				public void setVisible(Component component, boolean visible) {
+					if (!visible) {
+						super.remove(component);
```

Comment:
Needs to call `removeTab((AbstractPanel) component);` so the component is also removed from the internal tab list (it's a core bug, will open a PR to address that shortly).

Reply:
Thanks!

Reply:
Done

Reply:
Sorry, I meant that it should call `removeTab` instead of `remove` (this works fine but `removeTab` already calls `remove`).

Reply:
Hahaha ok ;) 2 sec

Reply:
Done++

Reply:
Thanks :)